src/cryptography/hazmat/backends/openssl/x509.py

```diff
@@ -235,6 +238,22 @@ def fingerprint(self, algorithm):
         h.update(der)
         return h.finalize()
 
+    def get_revoked_certificate(self, serial_number):
+        revoked = self._backend._ffi.new("X509_REVOKED **")
+        asn1_int = _encode_asn1_int_gc(self._backend, serial_number)
+        res = self._backend._lib.X509_CRL_get0_by_serial(
+            self._x509_crl, revoked, asn1_int
+        )
+        if res == 0:
+            return None
```

Comment:
Does this push an error? Do we need to clear the stack?

Reply:
No errors are pushed onto the stack in the test cases we have, but you'll never get me to say I'm 100% confident there's no scenario under which OpenSSL won't push an error onto it from some other function it calls.